transit_odp/organisation/tests/csv/test_service_codes.py

```diff
@@ -90,20 +109,18 @@ def test_missing_service_code():
     otc_lic1 = LicenceModelFactory(number=licence_number)
     ServiceModelFactory(
         licence=otc_lic1,
-        registration_number=otc_service_code,
+        registration_number=otc_registration_number,
         service_number=otc_service_number,
     )
 
     service_codes_csv = ServiceCodesCSV(org1.id)
 
     queryset = service_codes_csv.get_queryset()
-    assert queryset.count() == 1
-    row = service_codes_csv.get_row(queryset.first())
-    assert row[0] == ServiceCodesCSV.MISSING_DATA
-    assert row[1] == licence_number
-    assert row[2] == otc_service_code
-    assert row[3] == otc_service_number
-    assert row[4] == ""
+    row = queryset[0]
+    assert len(queryset) == 2
+    assert row["licence_number"] == licence_number
+    assert row["service_code"] == otc_registration_number
+    assert row["line_number"] == otc_service_number
```

Comment:
Same as above. In the `old` column `Service Status` can be `MISSING DATA` and `REGISTERED`.
Since this column doesn't exist anymore, it doesn't make sense keep this test case too!